src/transformers/models/whisper/modeling_whisper.py

```diff
@@ -1573,9 +1573,11 @@ def generate(
                     language_token = generation_config.language
                 elif generation_config.language in TO_LANGUAGE_CODE.keys():
                     language_token = f"<|{TO_LANGUAGE_CODE[generation_config.language]}|>"
+                elif generation_config.language in LANGUAGES.keys():
```

Comment:
Additionally, could we always lower-case the `language` argument, so that a user can pass `language=English` as well as `language=english`? Simply requires adding:
```diff
        if language is not None:
+           language = language.lower()
            generation_config.language = language
```
Just saw that this isn't supported currently in the modelling code, but is supported in the tokenizer (and also original repo):
https://github.com/huggingface/transformers/blob/57ffd8ab4c833e26b2288769f6031f94870a102c/src/transformers/models/whisper/tokenization_whisper.py#L399